frontend/src/app/component/phenopacket-list/phenopacket-list.component.ts

```diff
@@ -185,8 +185,10 @@ export class PhenopacketListComponent implements OnInit, OnDestroy {
               this.messageService.add({severity:'success', summary: "Phenopacket Upload Success!"});
 
             }, (error) => {
```

Comment:
At line173, the condition where the phenopacket id is already existing is somehow not found: we could replace 173 by the following lines : 
```typescript
currentPhenopacketsId.forEach(id => {
   if (newPhenopacket.id === id) {
      const errorMessage = `Phenopacket Id '${newPhenopacket.id}' already exists.`;
      throw new Error(errorMessage);
   }
}
```
Another thing: in that situation the error created here does not seem to be correctly caught in the error handling below and therefore won't display the error toast.

Reply:
Ah I wrote this in python format LOL